projects/ngneat/dialog/src/lib/specs/dialog.component.spec.ts

```diff
@@ -179,6 +180,107 @@ describe('DialogComponent', () => {
     });
   });
 
+  describe('when enableClose is set to onlyForLastOpenedDialog should call close only for last opened dialog', () => {
+    let dialogService: SpyObject<DialogService>;
+
+    beforeEach(() => {
+      spectator = createComponent(withConfig({ enableClose: 'onlyForLastOpenedDialog', id: 'close-last-open-dialog' }));
+      dialogService = spectator.inject(DialogService);
+
+      dialogService.dialogs.push(Object.assign(spectator.component.dialogRef, { id: 'close-last-open-dialog' }));
+    });
+
+    it('on escape as only dialog', () => {
```

Comment:
each it description should start with 'should'

Reply:
Text was copied from the other tests regarding the `enableClose` property, all tests have been updated to include `should (not) close ....`